jobs/pgbackup/spec

```diff
@@ -27,22 +27,21 @@ properties:
     default: pgdumpall
   pgbackup.host:
     description: "Hostname or IP address to connect to PostgreSQL on"
-    default: "localhost"
+    default: "127.0.0.1"
   pgbackup.port:
     description: "TCP port to connect to PostgreSQL on"
     default: 5432
   pgbackup.username:
     description: "User to connect to PostgreSQL as"
-    default: "pgbackup"
+    default: "PLEASE-CHANGE-THIS"
```

Comment:
what's the rationale for changing this default ?
what about removing the default instead ?